### PR TITLE
Test Branch

### PR DESCRIPTION
**Pull request by david-zou**
New file for Pull Requests

### CODE REVIEW

File: connectfour-test [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```
Comment (Ryxias) [inline connectfour-test]: What I meant when I said to "split out into a new file" was to keep all of the HTML in one file, separate from your Javascript (your _code_).  Then, your tests should be in a third, different file.

For example, you could have

`connect_four.html`, and `game.js`, and `test.html`

File: gameMechanics.js [hunk 2]
```diff
@@ -48,15 +48,21 @@ var insertDisc = function(currentGrid, selectedColumn, currentPlayer) {
...
-var stackCheck = function(input) {
+var connectCheck = function(input) {
 // stub function
 }
 
-var connectCheck = function(input) {
-// stub function
+var alternateTurns = function(currentTurn) {
+	var playerOne = 0;
+	var playerTwo = 1;
```
Comment (Ryxias) [inline gameMechanics.js]: I suggest using a Constant or an Enumeration to represent these values; it'll be less confusing to figure out whose turn it is.

File: gameMechanics.js [hunk 3]
```diff
@@ -101,4 +107,21 @@ var testInsertion = function (currentGrid, selectedColumn, playerFlag) {
...
 	var newGrid = insertDisc(currentGrid, selectedColumn, playerFlag);
 	printGrid(newGrid);
+}
+
+var testButton = function (currentGrid, selectedColumn, playerFlag) {
+	var newGrid = insertDisc(currentGrid, selectedColumn,playerFlag);
+	testPrint(newGrid);
+}
+
+var testPrint = function(inputGrid) {
```
Comment (Ryxias) [inline gameMechanics.js:290]: If you are looking for a method to change the contents of the grid without refreshing the entire page, you could draw a bunch of container elements to represent the grid, then use Javascript to change the contents of specific elements.

Alternatively, you can redraw the entire grid every turn.  It's much easier to code but less performant.

**Review by Ryxias — APPROVED** [review]

**Comment by david-zou** [comment]
Closing this PR to make another one with a clean slate.

**Pull request closed without merging**